dts/bindings/flash_controller/renesas,ra-flash-hp-controller.yaml

```diff
@@ -6,3 +6,17 @@ description: Renesas RA family flash high-performance controller
 compatible: "renesas,ra-flash-hp-controller"
 
 include: flash-controller.yaml
+
+properties:
+  block-32kb-linear-end:
+    type: int
+    required: true
+
+  block-32kb-dual-low-end:
+    type: int
+
+  block-32kb-dual-high-end:
+    type: int
+
+  reserved-area-num:
+    type: int
```

Comment:
@khoa-nguyen-18 : Could you please add description for these properties?

Reply:
I have just added the description for these properties, thank you. Can you please take a look?